src/WinRT.Runtime/Marshalers.cs

```diff
@@ -1181,9 +1194,23 @@ public static unsafe void CopyManaged(T value, IntPtr dest)
 #else 
     public
 #endif
-    static class MarshalInspectable<T>
+    static class MarshalInspectable<
+#if NET6_0_OR_GREATER || NET6_0

```

Comment:
Why is `NET6_0` used together with `_OR_GREATER`? It should be included.

Reply:
Seems to be working fine with the current .NET SDK version.  Addressed.